.github/workflows/linux_builds.yml

```diff
@@ -57,27 +58,29 @@ jobs:
           python --version
           scons --version
 
+      # Download, unzip and setup SwiftShader library [4466040]
+      - name: Download SwiftShader
+        run: |
+          wget https://github.com/qarmin/gtk_library_store/releases/download/3.24.0/swiftshader2.zip
+          unzip swiftshader2.zip
+          rm swiftshader2.zip
+          curr="$(pwd)/libvk_swiftshader.so"
+          sed -i "s|PATH_TO_CHANGE|$curr|" vk_swiftshader_icd.json
+
       # We should always be explicit with our flags usage here since it's gonna be sure to always set those flags
       - name: Compilation
         env:
           SCONS_CACHE: ${{github.workspace}}/.scons_cache/
         run: |
-          scons tools=yes tests=yes target=release_debug
+          scons tools=yes tests=yes target=release_debug module_mono_enabled=yes mono_glue=no debug_symbols=no
+          VK_ICD_FILENAMES=$(pwd)/vk_swiftshader_icd.json xvfb-run ./bin/godot.linuxbsd.opt.tools.64.mono --generate-mono-glue modules/mono/glue || true
```

Comment:
I think you don't need SwiftShader anymore, nor xvfb-run, you should be able to run Godot headless:
```
./bin/godot.linuxbsd.opt.tools.64.mono --headless --generate-mono-glue modules/mono/glue
```